Prace_domowe/Praca_domowa2/Dawid Janus/Praca Domowa 2.ipynb

```diff
@@ -0,0 +1,1627 @@
+{
```

Comment:
<p>Błąd należałoby wyliczyć tylko dla wartości imputowanych, bo gdy wyliczamy dla całej kolumny wynik jest zakłamany, ponieważ we wzorze jest wyliczana średnia, przez co liczność ma znaczenie.</p><p> </p><p>Prośba o zmodyfikowanie sposobu wyliczania błędu.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>